frontend/dockerfile/dockerfile_test.go

```diff
@@ -5508,6 +5508,7 @@ RUN echo hello
 }
 
 func testNamedImageContextTimestamps(t *testing.T, sb integration.Sandbox) {
+	integration.SkipIfDockerd(t, sb, "direct push")
```

Comment:
Looks like `TestNamedImageContextPlatform` is also failing; probably needs a skip added?

Reply:
Split commits in https://github.com/moby/buildkit/pull/3206 so you can cherry pick